api/services/recommended_app_service.py

```diff
@@ -95,7 +95,7 @@ def _fetch_recommended_apps_from_db(cls, language: str) -> dict:
 
             categories.add(recommended_app.category)  # add category to categories
 
-        return {'recommended_apps': recommended_apps_result, 'categories': list(categories)}
+        return {'recommended_apps': recommended_apps_result, 'categories': sorted(list(categories))}
```

Comment:
```suggestion
        return {'recommended_apps': recommended_apps_result, 'categories': sorted(categories)}
```
Simply sort the set, and the returned value is in list type. eg.
<img width="250" alt="image" src="https://github.com/langgenius/dify/assets/1935105/51597536-9ff5-4505-a7ca-ca6f32ddffc7">

Reply:
list(categories) duplicate removal, and sorted by it.

Reply:
Thanks. SGTM.